src/app/components/canvas/canvas.component.html

```diff
@@ -19,6 +19,7 @@
         matTooltip="Adjust Settings">
         <mat-icon>settings</mat-icon>
       </button>
+      <app-notification-dropdown [props]="{user:user, board:board}"></app-notification-dropdown>
```

Comment:
Small change but can we do something like `[user]="user" [board]="board"` instead?